apps/etterlatte-vilkaarsvurdering/src/main/resources/db/migration/V18__oppdaterer_vilkaar_oevrige_vilkaar.sql

```diff
@@ -0,0 +1,7 @@
+UPDATE delvilkaar SET beskrivelse = 'For at dette vilkåret skal være oppfylt, og gjenlevende ektefelle (§1-5 samboer likestilles) ha rett til ytelser etter dette kapitlet, må ett av disse vilkårene være oppfylt på tidspunktet for dødsfallet:
+
+a) var gift med den avdøde og ekteskapet hadde vart i minst fem år,
+b) har eller har hatt barn med den avdøde, eller
+c) har omsorg for barn under 18 år med minst halvparten av full tid (bestemmelsen gjelder også for barn som gjenlevende har med andre enn avdøde).
+
+Til en fraskilt person som helt eller i det vesentlige har vært forsørget av bidrag fra den avdøde, kan det ytes stønad etter dette kapitlet dersom ekteskapet varte i minst 25 år, eller minst 15 år hvis ektefellene hadde barn sammen.' WHERE vilkaar_type = 'OMS_OEVRIGE_VILKAAR';
```

Comment:
Er dette nødvendig?

Reply:
Fjerner og kjører de manuelt - se den andre pr for kommentar